### PR TITLE
Pre-DayCamp Tweaks

### PR DESCRIPTION
**Pull request by ElijahCaine**
So we just merged the Overhaul 2016 PR.  This PR will be where we make last minute tweaks to that update.  I kicked it off by adding live links for the content and slides.  Comment and make any changes you think need to be done before this Saturday.  We will be going through lesson 7 that day.

### CODE REVIEW

**Comment by Kennric** [comment]
Small issues:
Lesson 2, flavors of linux, Debian mis-spelled 'Deian'
Lesson 2, GNU/Linux - text wraps badly below the Tux image
Lesson 3, navigation concepts - the / in the item titles look strange: Flags / -v / --version is awkward, maybe "Flags (-v, --version)" instead?
Lesson 3: special characters: similar issue as above with the / - but now the character is first instead of the name, which is inconsistent
Lesson 4: what users can do - change "$ sudo su -         # use user password instead of root's" to $ sudo su -         # use your password instead of root's
Gneeral note: I don't see where we've explained that # means a comment, but we use it a lot in exambles and code blocks. Also, sometimes it's // in examples without explanation
Lesson 5: everything is a file, typo: "Well let’s say you’re programming an iterface" 
Lesson 5: do we ever explain what 'execute' permission is? read/write are more obvious, but execute maybe not, especially if we talk about permissions on directories
Lesson 5: we don't really talk about directories or permissions on them
Lesson 6: Popular Linux package managers: again the / in the item names are hard to parse

That's it from a brief scan. Overall, looking great.

**Comment by aaroncohen73** [comment]
You could add [this video](https://www.youtube.com/watch?v=D4fzInlyYQo) to the `Further Reading` section of `source/shell-navigation-os.rst` as a demonstration of what happens when you run `rm -rf /`

File: source/operating-systems.rst [hunk 1]
```diff
@@ -240,4 +256,15 @@ OSU Courses:
             - Building kernel modules
             - Memory management
 
-.. TODO: Add more Further Reading for independent students.
+Free Online Resources:
+    `OSDev.org`_  is a wiki dedicated to helping people develop their own
+    operating systems.  It's a big leap from this lesson, but great if you're
+    interested in learning the nitty-gritty.
+
+    `Operating Systems Design and Implementation`_ by Andrew S. Tenenbaum is a
```
Comment (aaroncohen73) [inline source/operating-systems.rst]: s/Tenenbaum/Tanenbaum/

File: source/users-groups-permissions.rst [hunk 2]
```diff
@@ -4,12 +4,14 @@
 Lesson 4: Users, Groups, Permissions
```
Comment (aaroncohen73) [inline source/users-groups-permissions.rst]: In `The User`, you should provide a couple examples of non-person users

Comment (aaroncohen73) [inline source/users-groups-permissions.rst]: At some point, the files `/etc/shadow` and `/etc/gshadow` should be mentioned, at least in passing (maybe around line 170 or so)

File: source/files.rst [hunk 3]
```diff
@@ -4,12 +4,14 @@
 Lesson 5: Files
```
Comment (aaroncohen73) [inline source/files.rst]: On line 56, you should provide a brief description of what an inode is

File: source/operating-systems.rst (hunk 1 above)
Comment (athai) [inline source/operating-systems.rst]: Done

File: source/operating-systems.rst [hunk 4]
```diff
@@ -4,12 +4,14 @@
 Lesson 2: Operating Systems
```
Comment (athai) [inline source/operating-systems.rst]: Done

Comment (athai) [inline source/operating-systems.rst]: Done

Comment (athai) [inline source/operating-systems.rst]: Done

File: source/first-steps.rst [hunk 5]
```diff
@@ -255,9 +273,9 @@ set.
...
 ---------------
 
-- `install Putty`_
-- `install Vagrant`_
-- `Virtual Machines`_
-- `install VirtualBox`_
+- More information on `Virtual Machines`_.
+- `install Putty`_ if you want to access a remote Linux box.
+- `install Vagrant`_ if you want to run a local Linux Virtual machine.
+- `install VirtualBox`_ in addition to Vagrant for local virtual machines.
```
Comment (athai) [inline source/first-steps.rst]: Done
